### DISCUSSION TITLE
Dataset SQL Expression Box in Metrics Tab Hard to Use

### DISCUSSION THREAD

**Discussion by jamesrayoub** [apache/superset#18554] [Ideas]
## Screenshot

![image](https://user-images.githubusercontent.com/31777200/151840456-ebd2e20f-5bf4-4b41-9f81-ff0c886aee82.png)

## Description

The SQL expression box is now very small and extremely difficult to work with... can we fix this so that it's expandable?

**Comment by geido** [comment]
Hello @jamesrayoub thanks for reporting this problem. I added a few labels to draw attention to this issue. In the meantime, you could use the legacy data editor that gives you more space to work with your SQL statements. Thank you!
